blocks/search.py

```diff
@@ -263,10 +263,19 @@ def search(self, input_values, eol_symbol, max_length,
             first iteration are ignored. This useful when the sequence
             generator was trained on data with identical symbols for
             sequence start and sequence end.
+        as_lists : bool, optional
+            If ``True``, a list of trimmed output sequences and a list of
+            their costs are returned, instead of the default output format.
 
         Returns
         -------
-        Sequences in the beam, masks, and corresponding log-probabilities.
+        outputs : :class:`numpy.ndarray`
+            The generated sequences, time is the first axis, the number in
+            the beam is the second.
+        mask : :class:`numpy.ndarray`
+            The mask for the outputs, same layout as `outputs`
```

Comment:
I don't think there is a format for it. I can't think of any NumPy function that would have a different number or different order of return values, and my guess is that they don't allow that (which is a reasonable thing to do...)

I agree with @dmitriy-serdyuk though, I think it's worthwhile to repeat. People are likely to look at the *Returns* section without having read every single parameter description.